packages/components/navbar/src/NavbarItem/NavbarItem.tsx

```diff
@@ -70,7 +70,7 @@ function _NavbarItem(
     >
       {icon && <NavbarItemIcon icon={icon} variant="secondary" />}
```

Comment:
We need to pass `isActive` rather than `variant`. Also, I think I'm seeing the old icons?

Reply:
Good catch, will have a look at updating the props.

Reply:
The icons you see in the web app are controlled from there, I assume we haven't updated them (Massao's migration work?)

Reply:
Example of using the v5 icons on the Space navigation 👍 

<img width="1574" alt="Screenshot 2024-06-07 at 14 21 12" src="https://github.com/contentful/forma-36/assets/103024358/0711a84e-e0a7-4597-8429-ee66d224ef98">